src/TextField/TextField.jsx

```diff
@@ -363,30 +362,6 @@ const TextField = React.createClass({
     return this.isMounted() ? this._getInputNode().value : undefined;
   },
 
-  setErrorText(newErrorText) {
-    warning(false, 'setErrorText() method is deprecated. Use the errorText property instead.');
-
-    if (this.isMounted()) {
-      this.setState({errorText: newErrorText});
-    }
-  },
-
-  setValue(newValue) {
```

Comment:
This components `clearValue()` method uses this function. We'll either have to remove that function too or implement it manually. Ideally we can just remove it but we never formally deprecated that function.